src/d/actor/d_a_tag_mk.cpp

```diff
@@ -7,18 +7,20 @@
 #include "dolphin/types.h"
 
 /* 00000078-00000084       .text getSwbit__10daTag_Mk_cFv */
-void daTag_Mk_c::getSwbit() {
+unsigned int daTag_Mk_c::getSwbit() {
     /* Nonmatching */
+    return this->mBase.mParameters & 0xff;
 }
 
 /* 00000084-00000090       .text getSwbit2__10daTag_Mk_cFv */
-void daTag_Mk_c::getSwbit2() {
+unsigned int daTag_Mk_c::getSwbit2() {
     /* Nonmatching */
+    return this->mBase.mParameters & 0xff;
 }
 
 /* 00000090-0000009C       .text getType__10daTag_Mk_cFv */
-void daTag_Mk_c::getType() {
-    /* Nonmatching */
+unsigned int daTag_Mk_c::getType() {
+    return this->mBase.mParameters & 0xff;
```

Comment:
Besides what Lago said, `this->mBase.mParameters` is most likely `fopAcM_GetParam(this)`